examples/destination/python/requirements.txt

```diff
@@ -0,0 +1,8 @@
+grpcio==1.62.1
```

Comment:
We should match the versions we are showing in the [development guide](https://github.com/fivetran/fivetran_sdk/blob/main/development-guide.md#versions). Let's use `1.61.1`?